liquibase-core/src/main/java/liquibase/changelog/ChangelogRewriter.java

```diff
@@ -15,6 +15,12 @@
 import java.util.regex.Pattern;
 
 public class ChangelogRewriter {
+
+    public static final String XSD_FILE_REGEX = "([dbchangelog|liquibase-pro])-3.[0-9]?[0-9]?.xsd";
```

Comment:
Do we need to take `*-latest` and version `4-` into account for any of these objects?

Reply:
This hasn't changed in the last two years. @arturobernalg just made the change of extracting this regular expression into a constant. I'm trying to understand whether this regular expression should be updated to include these versions (`4-` and `-latest`) or not.

Reply:
We seem to define a similar Pattern in multiple classes. In other places the regex is `dbchangelog-[\\w\\.]+.xsd` or `(?:-pro-|-)(?<version>[\\d.]*)\\.xsd`. Any chance we can use identical regex in all the different places? What about having the XSD pattern defined in one place?

Reply:
I asked @nvoxland about it, and he was pointing me out that changelogID is a changeset attribute that has been introduced in version `4-` so we should be checking to add this field (changelogID) for version 3-0 (not worth doing it also for earlier versions).